### PR TITLE
final merge develop to master

### PR DESCRIPTION
**Pull request by NKOKKILIGADDA87**

### CODE REVIEW

**Review by drtechie — CHANGES_REQUESTED** [review]
Please remove references to -next
@pavangsk please ensure this is applied to all UI repos.

Add proper description for the PR.

File: src/environments/environment.ci.ts.template [hunk 1]
```diff
@@ -215,34 +215,33 @@ export const environment = {
...
-  downloadBabyDeathReportURL: `${ECD_API}ecdReportController/getECDBabyDeathReport`,
-  downloadNotConnectedReportURL: `${ECD_API}ecdReportController/getECDNotConnectedPhonelistDiffformatReport`,
-  downloadJsyReportURL: `${ECD_API}ecdReportController/getECDJSYRelatedComplaintsReport`,
-  downloadMiscarriageReportURL: `${ECD_API}ecdReportController/getECDMiscarriageReport`,
-
-};
+  getGenderMasterUrl:`${ECD_API}master/getGender`,
+  
+// Reports 
+downloadCumulativeReportURL:`${ECD_API}ecdReportController/getECDCumulativeDistrictReport`,
```
Comment (drtechie) [inline src/environments/environment.ci.ts.template]: Indent is off on this file

**Comment by pavangsk** [comment]
> Please remove references to -next @pavangsk please ensure this is applied to all UI repos.
> 
> Add proper description for the PR.

**Comment by pavangsk** [comment]
Hi @drtechie Informed team to remove -next in all repositories, Team will update the changes and raise new PR to Main branch.

**Review by helenKaryamsetty — COMMENTED** [review]
THere are merge conflicts in this PR please resolve those

**Pull request merged**